client/src/components/Tabs/BasicTabs.js

```diff
@@ -0,0 +1,28 @@
+import React, { useEffect, useState } from "react";
+import { Tabs } from "antd";
+
+const { TabPane } = Tabs;
+
+const BasicTabs = ({ tabData }) => {
+  const { defaultView, position, tabs } = tabData;
+  return (
+    <Tabs tabPosition={position} defaultActiveKey={defaultView}>
+      {tabs.map((e, index) => {
+        if (e.display) {
+          return (
+            <TabPane tab={e.tabName} key={index}>
```

Comment:
you can remove the if block if you do disabled={!e.display}

Reply:
done!